docs/how-to-guides/software-management/software-support.md

```diff
@@ -49,4 +67,23 @@ Prioritization meetings are typically a loosely guided discussion in which the t
 1. Ask for any closing remarks
 
 ??? Note "Issue deferral"
-    Deferring an issue generally means "this is a good, important issue, but it requires further discussion." This discussion should not interrupt the flow of prioritization, and discussing deferred issues is usually a good final topic.
\ No newline at end of file
+    Deferring an issue generally means "this is a good, important issue, but it requires further discussion." This discussion should not interrupt the flow of prioritization, and discussing deferred issues is usually a good final topic.
+
+
+## Continous Support 
+
+An additional 2 weeks per quarter are dedicated for handling small issues. We refer to this as continious support, as it's suppport time between dedicated sprints. This amounts to roughly 6 hours a week to stay on top of GitHub notifications.  
```

Comment:
2 week per what? For someone external I would just say, we allocate X hours or X FTE per quarter that stuff happens in.

Reply:
Also, is this much information (task levels per dev how many hours) really necessary? What is the rationale for giving this much insight? (I ask because external folks may assess the speed with which they get a response with the perception that someone has 6 hours per week without knowing all of the context.)

Reply:
> 2 week per what? For someone external I would just say, we allocate X hours or X FTE per quarter that stuff happens in.

I can probably change this to "roughly 6 hours a week between support sprints."

> Also, is this much information (task levels per dev how many hours) really necessary?

Writing down the expectations for continuous support was one of the action items that came out of the last retro. Originally, I was going to make it internal, but couldn't see the harm of including it here. I usually big on transparency, but if there is reason it might be a problem in this context I'm open to moving it internal.

Reply:
updated this, and moved most of the original text into the internal docs.